packages/transitive-overlay/src/transitive-styles.js

```diff
@@ -21,7 +21,7 @@ STYLES.places = {
       : "none",
   fill: (display, place) => {
     if (coreUtils.map.isBikeshareStation(place)) {
-      return "#f00";
+      return "#E60000"; /* from #f00 to meet WCAG */
```

Comment:
We probably don't need a comment about the previous value here.

Reply:
Removed in 1f02f4c26f30f25588f4519499764a86dcf4bfc5